examples_tests/50.RadixSort/Histogram.comp

```diff
@@ -24,31 +24,26 @@ layout(push_constant) uniform pushConstants
     layout (offset = 0) uint shift;
 } u_push_constants;
 
-shared uint local_histogram[NUM_BUCKETS * WG_DIM];
+shared uint local_histogram[NUM_BUCKETS * WG_DIM]; // 16KB
 
 void main()
 {
 	for (int i = 0; i < NUM_BUCKETS; i++)
 		local_histogram[i * WG_DIM + gl_LocalInvocationID.x] = 0;
 
+	// Todo: Probably remove memoryBarrier from here?
```

Comment:
you can replace the whole of `main` with about 5 lines which include `nbl_glsl_workgroupBallot*`